warehouse/query-core/src/main/java/datawave/query/tables/ShardQueryLogic.java

```diff
@@ -445,6 +459,8 @@ public void initialize(ShardQueryConfiguration config, Connector connection, Que
         
         config.setQueryString(getQueryPlanner().getPlannedScript());
         
+        this.setQueryPlan(getQueryPlanner().getPlannedScript());
```

Comment:
does this not simply set the query plan on the configuration?

Reply:
No, this sets the query plan onto a field i added into the logic. If the configuration fails, the plan will be set into this field. If the configuration initialization is successful, I just assigned it to both fields which isn't completely necessary, but at least it keeps them the same. It could be removed at this point since its already set in the configuration.